src/main/resources/META-INF/persistence.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <persistence version="1.0" xmlns="http://java.sun.com/xml/ns/persistence" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_1_0.xsd">
-    <persistence-unit name="VDCNet-ejbPU" transaction-type="JTA">
+    <persistence-unit name="VDCNet-ejbPU-1" transaction-type="JTA">
```

Comment:
Can we pick a better name for this rather than just -1

Reply:
Renamed to masterPU.